parent/pom.xml

```diff
@@ -1775,58 +1776,62 @@
                 <infinispan.test.jgroups.protocol>tcp</infinispan.test.jgroups.protocol>
             </properties>
         </profile>
         <profile>
             <id>test-functional</id>
             <properties>
-                <defaultTestGroup>functional</defaultTestGroup>
+                <defaultTestNGGroups>functional</defaultTestNGGroups>
                 <infinispan.test.jgroups.protocol>tcp</infinispan.test.jgroups.protocol>
             </properties>
         </profile>
         <profile>
             <id>smoke</id>
             <properties>
-                <defaultTestGroup>smoke</defaultTestGroup>
+                <defaultTestNGGroups>smoke</defaultTestNGGroups>
+                <defaultJUnitGroups>org.infinispan.commons.test.categories.Smoke</defaultJUnitGroups>
                 <infinispan.test.jgroups.protocol>tcp</infinispan.test.jgroups.protocol>
             </properties>
         </profile>
         <profile>
             <id>test-unstable</id>
             <properties>
-                <defaultTestGroup>unstable</defaultTestGroup>
-
-                <defaultExcludedTestGroup/>
+                <defaultTestNGGroups>unstable</defaultTestNGGroups>
+                <defaultJUnitGroups>org.infinispan.commons.test.categories.Unstable</defaultJUnitGroups>
+                <defaultExcludedTestNGGroups/>
+                <defaultExcludedJUnitGroups/>
                 <infinispan.test.jgroups.protocol>tcp</infinispan.test.jgroups.protocol>
             </properties>
         </profile>
         <profile>
             <id>test-stress</id>
             <properties>
-                <defaultTestGroup>stress</defaultTestGroup>
-                <defaultExcludedTestGroup/>
+                <defaultTestNGGroups>stress</defaultTestNGGroups>
+                <defaultJUnitGroups>org.infinispan.commons.test.categories.Stress</defaultJUnitGroups>
+                <defaultExcludedTestNGGroups/>
+                <defaultExcludedJUnitGroups/>
                 <infinispan.test.parallel.threads>1</infinispan.test.parallel.threads>
                 <infinispan.test.jgroups.protocol>tcp</infinispan.test.jgroups.protocol>
             </properties>
         </profile>
         <profile>
             <id>test-unit</id>
             <properties>
-                <defaultTestGroup>unit</defaultTestGroup>
+                <defaultTestNGGroups>unit</defaultTestNGGroups>
             </properties>
         </profile>
         <profile>
             <id>test-jgroups</id>
             <properties>
-                <defaultTestGroup>jgroups</defaultTestGroup>
+                <defaultTestNGGroups>jgroups</defaultTestNGGroups>
             </properties>
         </profile>
         <profile>
             <id>test-transaction</id>
             <properties>
-                <defaultTestGroup>transaction</defaultTestGroup>
+                <defaultTestNGGroups>transaction</defaultTestNGGroups>
```

Comment:
These profiles seem targeted to the core module, maybe we should move them there.

Reply:
Let's leave them here for now, as they do no harm.